### ISSUE TITLE
alelle frequency filter  is not properly

### ISSUE THREAD

**Issue by vinod1981** [frankvogt/vcf2gwas#22]
It seems, allele frequency filter in vcf2gwas doesn't work properly, I am setting it q 0.02 but I can still see many SNPs with less than 0.02 MAF in final .assoc.txt files. Even I tried other filters too, and in my case it doesn't work.
Please see it and let me know if I am doing something wrong.

final command:
`vcf2gwas -T 4 -v GT_1.vcf.gz -pf Leaf_Soil_AB_All.csv -ap -cf pca_updated_5.csv -ac -lmm -q 0.02 -o vcf2gwas/ortho_dist/
`
Thanks,

**Comment by vinod1981** [comment]
Dear Frank,
I am using this tool since long time and observed this issue just few days back when I started analyzing my data for postGWAS analysis. I used many files and many af cutoff with -q option but the fact is that the tool is not removing them from the final gwas results '.assoc.txt' and they are making a lot of noise in the results. So, now I am removing them manually after the gwas analysis is completed with vcf2gwas.
Thanks,
